single_tech_samples/cosmosdb/infrastructure/azuredeploy.cosmosdb.json

```diff
@@ -85,49 +85,49 @@
 			"minValue": 400,
 			"maxValue": 1000000,
 			"metadata": {
-				"description": "The throughput for the database to be shared"
+				"description": "The throughput for the database to be shared."
 			}			
 		},
 		"sharedContainer1Name": {
 			"type": "string",
 			"defaultValue": "container1",
 			"metadata": {
-				"description": "The name for the first container with shared throughput"
+				"description": "The name for the first container with shared throughput."
 			}
 		},
 		"sharedContainer1PartitionKey": {
 			"type": "string",
 			"defaultValue": "/Container1Id",
 			"metadata": {
-				"description": "The partition key for the first container with shared throughput"
+				"description": "The partition key for the first container with shared throughput."
 			}
 		},
 		"sharedContainer2Name": {
 			"type": "string",
 			"defaultValue": "container2",
 			"metadata": {
-				"description": "The name for the second container with shared throughput"
+				"description": "The name for the second container with shared throughput."
 			}
 		},
 		"sharedContainer2PartitionKey": {
 			"type": "string",
 			"defaultValue": "/Container2Id",
 			"metadata": {
-				"description": "The partition key for the second container with shared throughput"
+				"description": "The partition key for the second container with shared throughput."
 			}
 		},
 		"sharedContainer3Name": {
 			"type": "string",
 			"defaultValue": "container3",
 			"metadata": {
-				"description": "The name for the third container with shared throughput"
+				"description": "The name for the third container with shared throughput."
 			}
 		},
 		"sharedContainer3PartitionKey": {
 			"type": "string",
 			"defaultValue": "/Container3Id",
 			"metadata": {
-				"description": "The partition key for the third container with shared throughput"
+				"description": "The partition key for the third container with shared throughput."
```

Comment:
@tomomi-code, quick suggestion for the arm template for Cosmos. Perhaps we could use a copy statement to create the containers and reduce the number of parameters.

Reply:
thanks @jsburckhardt 
copy statement sounds better. @tomomi-code I can update to use a copy statement. shall we made another pr, or I will just commit to this branch?

Reply:
@jsburckhardt @siliang-j-1225 
Thanks for the comments, but I think this change is out of the scope of this PR.

I think its best to create a new task and PR to make this change.